contracts/InternalOptimisticOracle.sol

```diff
@@ -103,32 +140,37 @@ contract InternalOptimisticOracle is Testable {
 
         request.disputer = msg.sender;
 
-        // If the final fee gets updated between the time the request is made and the time the dispute is made, the
+        // If the final fee gets increased between the time the request is made and the time the dispute is made, the
```

Comment:
I might have been wrong in my previous comments on this edge case as rereading OO I see that `finalFee` gets stored on price request.

Reply:
I think you were right last time when you thought about this:
In  this contract we have a local [RequestPrice](https://github.com/UMAprotocol/dev-quickstart/blob/c5d8a9148546e641d289cafd06e80bd8f434c35e/contracts/InternalOptimisticOracle.sol#L27) object instantiated when we do the local `requestPrice`. During this call we store the final fee.

If the `Store.finalFee` is updated before anyone calls `OA.proposePrice` or `OA.disputePrice` the local `OA.request.finalFee` would be outdated and then won't work when interacting with the OO. Does it make sense?